tests/method-chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -6,10 +157,7 @@ exports[`test test.js 1`] = `
 ({}.a().b());
 ({}).a().b();
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-method()
-  .then(x => x)
-  [\"abc\"](x => x)
-  [abc](x => x);
+method().then(x => x)[\"abc\"](x => x)[abc](x => x);
```

Comment:
I'm not sure how representative it is. In practice, there's likely going to be a function with `{}` which will trigger everything to expand. I'm not sure we want to always automatically expand chained expressions. I think we should let the 80 column algorithm do its work and if it doesn't fit in one line then expand.